docs/get-started/getting-started-with-rapids-accelerator-on-databricks.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: page
+title: Databricks
+nav_order: 3
+parent: Getting-Started
+---
+
+# Getting started with RAPIDS Accelerator on Databricks
+This guide will run through how to set up the RAPIDS Accelerator for Apache Spark 3.0 on Databricks. At the end of this guide, the reader will be able to run a sample Apache Spark application that runs on NVIDIA GPUs on Databricks.
+
+## Prerequisites
+* Apache Spark 3.0 running in DataBricks Runtime 7.0 ML with GPU 
+    * AWS: 7.0 ML (includes Apache Spark 3.0.0, GPU, Scala 2.12)
+    * Azure: 7.0 ML (GPU, Scala 2.12, Spark 3.0.0)
+
+The number of GPUs per node dictates the number of Spark executors that can run in that node. Each executor should only be allowed to run 1 task at any given time.
+
+## Start a Databricks Cluster
+Create a Databricks cluster by going to Clusters, then clicking “+ Create Cluster”. Ensure the cluster meets the prerequisites above by configuring it as follows:
+1. On AWS, make sure to use 7.0 ML (includes Apache Spark 3.0.0, GPU, Scala 2.12), or for Azure, choose 7.0 ML (GPU, Scala 2.12, Spark 3.0.0).
+2. Under Autopilot Options, disable auto scaling.
+3. Choose the number of workers that matches the number of GPUs you want to use.
+4. Select a worker type. On AWS, use nodes with 1 GPU each such as p3.xlarge or g4dn.xlarge. p2 nodes do not meet the architecture requirements for the Spark worker (although they can be used for the driver node). For Azure, choose GPU nodes such as Standard_NC6s_v3

```

Comment:
Should we mention something about the type of node and cost.  That we recommend the X type with T4 GPUs for cost savings.

Reply:
I would want to avoid that. But we have updated the ETL benchmark with T4 costs as a guidance.